README.md

```diff
@@ -21,7 +21,7 @@ Generate all the Things! Visit [our website](https://jacquescarette.github.io/Dr
 
 ## What is Drasil?
 
-For well understood domains, building software ought to be a matter of engineering, based on solid scientific foundations. The ultimate test of "well understood" is being able to teach the domain language to a computer.  Drasil is a framework for generating all of the software artifacts for (well understood) research software, from the natural knowledge base of the domain.
+For well understood domains, building software ought to be a matter of engineering, based on solid scientific foundations. The ultimate test of "well understood" is being able to teach the domain language to a computer.  Drasil is a framework for generating all of the software artifacts for (well understood) research software, from the natural knowledge base of the domain. The new position's Well-Understood paper is at [Well-Understood Paper](https://github.com/JacquesCarette/Drasil/blob/master/Papers/WellUnderstood/wu.pdf).
```

Comment:
that is very tortured English. Also modern web linking has link text flow in the language itself rather than highlighting that there is a link.